api/flaskr/routes/woocommerce.py

```diff
@@ -0,0 +1,94 @@
+from flask import Blueprint, json, request, url_for, redirect, session, current_app, jsonify
+from flask.wrappers import Response
+from ..wcapi import woocommerce
+
+woocommerce_route = Blueprint('woocommerce_route', __name__)
+
+# Products
+@woocommerce_route.route("/createProduct", methods=['POST'])
```

Comment:
Why .route ?

Reply:
isn't it the way to add to add routes to a blueprint ?

Reply:
I don't quite remember TBH.